Fix test_cookie_tracker_detection in Firefox

Currently [failing on master in Firefox](https://travis-ci.org/EFForg/privacybadger/jobs/210598442). Was disabled when [Selenium tests in Firefox were merged](https://github.com/EFForg/privacybadger/commit/e517a49), re-enabled as part of https://github.com/EFForg/privacybadger/pull/1205 (`git bisect` seemed to lead me to 936a1f8908a8ff2b3c784d8e6a86b05e4210eaaa), but isn't working still:
```
________________________________ CookieTest.test_cookie_tracker_detection ________________________________
...
>               window_utils.close_windows_with_url(self.driver, self.popup_url)
selenium/cookie_test.py:77: 
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _

...
>       raise Exception("was not able to find window for url " + url)
E       Exception: was not able to find window for url moz-extension://37702c80-1338-4638-b07a-33e87fa5327d/skin/popup.html
selenium/window_utils.py:54: Exception

--- a/src/heuristicblocking.js
+++ b/src/heuristicblocking.js
@@ -475,7 +475,7 @@ var extractCookieString = function(details) {
 
   for (var i = 0; i < headers.length; i++) {
     var header = headers[i];
-    if (header.name == "Cookie" || header.name == "Set-Cookie" ) {
+    if (header.name.toLowerCase() == "cookie" || header.name.toLowerCase() == "set-cookie" ) {
       if (!cookies) {
         cookies = header.value;
       } else {

--- a/tests/selenium/cookie_test.py
+++ b/tests/selenium/cookie_test.py
@@ -74,7 +74,6 @@ class CookieTest(pbtest.PBSeleniumTest):
                     print("Popup UI has been updated. Yay!")
                     break
                 window_utils.close_windows_with_url(self.driver, self.popup_url)
-                window_utils.close_windows_with_url(self.driver, self.popup_url)
                 print("popup UI has not been updated yet. try again in 10 seconds")
                 time.sleep(10)
 

--- a/tests/selenium/cookie_test.py
+++ b/tests/selenium/cookie_test.py
@@ -20,9 +20,9 @@ class CookieTest(pbtest.PBSeleniumTest):
     def assert_pass_opera_cookie_test(self, url, test_name):
         self.load_url(url)
         self.assertEqual(
-                "PASS",
-                self.js("return document.getElementById('result').innerHTML"),
-                "Cookie test failed: %s" % test_name)
+            "PASS",
+            self.js("return document.getElementById('result').innerHTML"),
+            "Cookie test failed: %s" % test_name)
 
     def test_should_pass_std_cookie_test(self):
         self.assert_pass_opera_cookie_test("https://gistcdn.githack.com/gunesacar/79aa14bac95694d38425d458843dacd6/raw/3d17cc07e071a45c0bf536b907b6848786090c8a/cookie.html",  # noqa
@@ -67,15 +67,15 @@ class CookieTest(pbtest.PBSeleniumTest):
         self.load_url(SITE1_URL)
         window_utils.close_windows_with_url(self.driver, SITE3_URL)
         for i in range(60):
-                self.load_pb_ui(SITE1_URL)
-                self.get_tracker_state()
+            self.load_pb_ui(SITE1_URL)
+            self.get_tracker_state()
 
-                if THIRD_PARTY_TRACKER in self.cookieBlocked:
-                    print("Popup UI has been updated. Yay!")
-                    break
-                window_utils.close_windows_with_url(self.driver, self.popup_url)
-                print("popup UI has not been updated yet. try again in 10 seconds")
-                time.sleep(10)
+            if THIRD_PARTY_TRACKER in self.cookieBlocked:
+                print("Popup UI has been updated. Yay!")
+                break
+            window_utils.close_windows_with_url(self.driver, self.popup_url)
+            print("popup UI has not been updated yet. try again in 10 seconds")
+            time.sleep(10)
 
         self.assertTrue(THIRD_PARTY_TRACKER in self.cookieBlocked)
 
@@ -119,33 +119,33 @@ class CookieTest(pbtest.PBSeleniumTest):
         self.cookieBlocked = {}
         self.blocked = {}
         try:
-                clickerContainer = self.driver.find_element_by_class_name("clickerContainer")
-                self.assertTrue(clickerContainer)
+            clickerContainer = self.driver.find_element_by_class_name("clickerContainer")
+            self.assertTrue(clickerContainer)
         except:
-                print("no action state information was found")
-                return
+            print("no action state information was found")
+            return
 
         tooltips = clickerContainer.find_elements_by_xpath("//*[contains(@class,'clicker tooltip')]")
         for t in tooltips:
-                origin = t.get_attribute('data-origin')
+            origin = t.get_attribute('data-origin')
 
-                # assert that this origin is never duplicated in the UI
-                self.assertTrue(origin not in self.nonTrackers)
-                self.assertTrue(origin not in self.cookieBlocked)
-                self.assertTrue(origin not in self.blocked)
+            # assert that this origin is never duplicated in the UI
+            self.assertTrue(origin not in self.nonTrackers)
+            self.assertTrue(origin not in self.cookieBlocked)
+            self.assertTrue(origin not in self.blocked)
 
-                action_type = t.get_attribute('data-original-action')
-                if action_type == 'allow':
-                    self.nonTrackers[origin] = True
-                elif action_type == 'noaction':
-                    self.nonTrackers[origin] = True
-                elif action_type == 'cookieblock':
-                    self.cookieBlocked[origin] = True
-                elif action_type == 'block':
-                    self.blocked[origin] = True
-                else:
-                    print("what is this?!? " + str(action_type))
-                    self.assertTrue(False)
+            action_type = t.get_attribute('data-original-action')
+            if action_type == 'allow':
+                self.nonTrackers[origin] = True
+            elif action_type == 'noaction':
+                self.nonTrackers[origin] = True
+            elif action_type == 'cookieblock':
+                self.cookieBlocked[origin] = True
+            elif action_type == 'block':
+                self.blocked[origin] = True
+            else:
+                print("what is this?!? " + str(action_type))
+                self.assertTrue(False)
 
 
 if __name__ == "__main__":

